src/elm/Auth.elm

```diff
@@ -146,87 +147,107 @@ maybePrivateKey model =
 
 -- VIEW
 
+{-| You're gonna want a comment explaining how this works I reckon. -}
+type alias Handlers msg =
```

Comment:
Yeah, looks like you are changing the way we map Msgs. Maybe we can talk in more depth about this on a video call